huaweicloud/services/elb/data_source_huaweicloud_elb_certificate.go

```diff
@@ -58,7 +58,7 @@ func dataSourceELbCertificateV3Read(_ context.Context, d *schema.ResourceData, m
 	config := meta.(*config.Config)
```

Comment:
config is same with the package name, it is recommended to rename it.

Reply:
done